app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+	def create
+		# byebug
+		@comment = current_user.comments.new(body: params[:comment][:body], commentable_type: params[:comment][:commentable_type], commentable_id: params[:comment][:commentable_id])
+		if @comment.save
+			flash[:success] = "Comment successfully created"
+			redirect_to :controller => 'home', :action => 'index'
```

Comment:
why not just: `redirect_to home_path` ?
and have a show action in **HomesController** let me know if it doesn't work that way